frankenphp.go

```diff
@@ -294,6 +297,9 @@ func Init(options ...Option) error {
 		return NotEnoughThreads
 	}
 
+	// create enough request handles to keep spin-lock down
+	requestHandles = ConcurrentHandle.NewConcurrentHandle(opt.numThreads * 4)
```

Comment:
Since this uses atomics + spin lock, we use `*4` number of "slots" to give us a pretty high chance of getting a lock on a particular slot.